publish-version.sh

```
@@ -0,0 +1,22 @@
+
+# Run the prepublish setup script
+npm run prepublish:setup
+
+# Check if dist directory exists
+if [ ! -d "dist" ]; then
+  echo "Error: dist directory not found. The prepublish:setup script may have failed."
+  exit 1
+fi
+
+# Change directory to the dist folder
+cd dist
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)